config/webpack.config.cozy-collect.js

```diff
@@ -14,8 +15,14 @@ module.exports = {
   },
   plugins: [
     new ProvidePlugin({
-      'initKonnectors': 'expose?initKonnectors!config/konnectors.json',
-      'initFolders': 'expose?intFolders!config/folders.json'
-    })
+      initKonnectors: 'expose-loader?initKonnectors!config/konnectors.json',
+      initFolders: 'expose-loader?intFolders!config/folders.json'
```

Comment:
What `initFolders` is used for ?

Reply:
Good question, it seems to be used by the Collect store but I'll take a look on it 👍